sonar-core/src/main/java/org/sonar/core/user/UserDto.java

```diff
@@ -88,11 +97,39 @@ public String getScmAccounts() {
     return scmAccounts;
   }
 
-  public UserDto setScmAccounts(@Nullable String scmAccounts) {
-    this.scmAccounts = scmAccounts;
+  public List<String> getScmAccountsAsList() {
```

Comment:
This method should replace getScmAccounts(), so getScmAccounts() should not be public as it's only used to test encodeScmAccounts().